jobs/resource_job.go

```diff
@@ -342,7 +356,8 @@ type JobRun struct {
 	Trigger     string   `json:"trigger,omitempty"`
 	RuntType    string   `json:"run_type,omitempty"`
 
-	OverridingParameters RunParameters `json:"overriding_parameters,omitempty"`
+	OverridingParameters RunParameters  `json:"overriding_parameters,omitempty"`
+	JobParameter         []JobParameter `json:"job_parameters,omitempty"`
```

Comment:
The field name needs to stay plural.
```suggestion
	JobParameters        []JobParameter `json:"job_parameters,omitempty"`
```